src/pages/en/guides/imports.md

```diff
@@ -135,6 +137,48 @@ It can also be useful to place images in the `public/` folder as explained on th
 Adding **alt text** to `<img>` tags is encouraged for accessibility! Don't forget to add an `alt="a helpful description"` attribute to your image elements. You can just leave the attribute empty if the image is purely decorative.
 :::
 
+## `Astro.glob()`
+
+[`Astro.glob()`](/en/reference/api-reference/#astroglob) is a way to import many files at once.
+
+`Astro.glob()` only takes one parameter: a relative [glob pattern](/#globpatterns) matching the local files you'd like to import. It’s asynchronous, and returns an array of each matching file's exports.
+
+```astro title="src/components/my-component.astro"
+---
+// imports all files that end with `.md` in `./src/pages/post/`
+const posts = await Astro.glob('../pages/post/*.md'); 
+---
+<!-- Renders an <article> for the first 5 blog posts -->
+<div>
+{posts.slice(0, 4).map((post) => (
+  <article>
+    <h1>{post.frontmatter.title}</h1>
+    <p>{post.frontmatter.description}</p>
+    <a href={post.frontmatter.url}>Read more</a>
+  </article>
+))}
+</div>
+```
+
+### Glob Patterns
+
+A glob pattern is a file path that supports special wildcard characters. This lets it reference multiple files. For example,
+
+`pages/**/*.{md,mdx}` starts within the pages subdirectory, looks through all of its subdirectories (`/**`), and matches any filename (`/*`) that ends in either `.md` or `.mdx` (`.{md,mdx}`). 
```

Comment:
```suggestion
`./pages/**/*.{md,mdx}` starts within the pages subdirectory, looks through all of its subdirectories (`/**`), and matches any filename (`/*`) that ends in either `.md` or `.mdx` (`.{md,mdx}`).

Glob patterns must begin with either `./` (to start in the current directory) or `/` (to start at the root of the project).
 
```
My mistake - Globs have to start with either `/` or `./`

Reply:
I'm not sold on the "for example" usage here.

I think either "For example, ... " starting the sentnce in the same line. It would be one thing if the next thing were a full code snippet, but it's not, it's just a sentence.

Reply:
no objections to the text, though!